### PR TITLE
Horizontal Nav is outdated and inconsistent with vertical navigation #387

### PR DESCRIPTION
**Pull request by deernano**
## Description

* The horizontal navigation is outdated and inconsistent with the latest vertical navigation. This PR is to update the overview and design, also the pictures of the horizontal navigation to the latest version.
* Related github PR: https://github.com/patternfly/patternfly-design/issues/387
* Related Jira story: https://patternfly.atlassian.net/browse/PTNFLY-2525

## Changes

* Change 01 over.md
* Change 02 design.md
* Change 03 img

## Contribution Submission Checklist

- [X] A GitHub issue was created and reviewed with the PatternFly community.
- [X] The pattern design overview.md and design.md templates were used to provide all information, relevant to this design.
- [X] Any new files have been added to the folder for that pattern.
- [X] The folder, for brand new patterns, has been titled with the name of the pattern.
- [X] All images are in an "img" folder and embedded images have the correct filepath.
- [X] The overview.md, design.md and site.md files, for this design have been commited and pushed to the patternfly-design repository, for review.

### CODE REVIEW

**Comment by deernano** [comment]
@mcarrano  please reivew this pr , thank you~

File: pattern-library/navigation/horizontal-navigation/design/overview.md [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
 
 ## Navbar Variations
 
 ### Single-Level Menu Bar
-![Single Level Menu Bar](img/single-level-menu-bar.png)
+![Single Level Menu Bar](img/masthead-horizontal@2x.png)
 
 ### Two-Level Menu Bar
-![Two Level Menu Bar](img/two-level-menu-bar.png)
+![Two Level Menu Bar](img/menu-bar-two-level.png)
```
Comment (mcarrano) [inline pattern-library/navigation/horizontal-navigation/design/overview.md]: Can we update the mockups here to remove the drop-down indicators from the secondary menu? We don't want to support this going forward as it implies a third level.

File: pattern-library/navigation/horizontal-navigation/design/design.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-![Horizontal Navbar with Single-Level](img/horizontal-nav-single.png)
+![Horizontal Navbar with Single-Level](img/design-single-level-nav@2x.png)
 
 The Single-Level Menu Bar is the default navbar, all other options are added to this basic pattern.
 
 1. **Menu Item - Selected**: The current selection will be highlighted to indicate the current page. There will be a current selection at all times. When an application is opened, the first (left-most) item will be selected by default.
 2. **Menu Item - On-Hover**: Hover state provides feedback that menu items are active.
 
-
 ## Two-Level Menu Bar
```
Comment (mcarrano) [inline pattern-library/navigation/horizontal-navigation/design/design.md]: Also remove the drop-down indicators from the mockup here.

File: pattern-library/navigation/horizontal-navigation/design/design.md [hunk 3]
```diff
@@ -24,11 +24,9 @@ When a second level of persistent navigation is needed, add a second tier of lin
...
+Add a drop-down menu to a single menu bar to expose a vertical list of sub-categories that the user can select from. Note that when drop-down menus are used, all categories are not required to have secondary navigation. The drop-down menus only support up to two levels of navigation. If you need the tertiary navigation, [Vertical Navigation](http://www.patternfly.org/pattern-library/navigation/vertical-navigation/) is the preferred choice over horizontal navigation.
 
 1. **Primary Menu Item**: Any primary item that supports a secondary level of navigation will add a down-caret icon to indicate that a drop-down menu exists. The primary item will display its selected state when any secondary page within that branch is selected. This will provide the user feedback about which primary section they are working in. When an application is initially opened, the first secondary page within the first section will be opened, by default.
 2. **Drop-Down Menu**: Single-clicking a primary item opens the drop down. Clicking anywhere outside of the menu will dismiss it.
```
Comment (mcarrano) [inline pattern-library/navigation/horizontal-navigation/design/design.md]: While we do want to remove the fly-out menu here (item 4), it's still worth showing the there is a hover state for items in the drop-down menu (first 2 sentences of item 3)

**Comment by deernano** [comment]
@mcarrano Updated all the pictures and changed the third description of Menu Bar with Drop-Downs in the design.md

**Comment by mcarrano** [comment]
Looks great @deernano Thanks for making these changes. @LHinson please also review and merge if you don't see any issues.

**Comment by LHinson** [comment]
Thanks @deernano!

**Pull request merged**